install.sh

```diff
@@ -247,8 +247,29 @@ EOF
 
 # Change the rights to the Pi user
 cd "${BASEDIR}/"
+
+# Updates for docker support
+# Create data directory
+mkdir -p data
```

Comment:
Does this error if the directory already exists? Does it matter..
```suggestion
mkdir -p data 2>/dev/null
```

Reply:
Does not matter. It will not produce an error if exists.. I tried :)